mobilenet/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tensorflow-models/mobilenet",
-  "version": "0.2.2",
+  "version": "0.3.0",
```

Comment:
hey Ann, we should up this to 1.0.0 (all models except speech-commands). Also can you update the demo.html to depend on 1.0.0 instead of alpha5. Thanks!